.test-defs/RegisterSeed.yaml

```diff
@@ -0,0 +1,30 @@
+kind: TestDefinition
+metadata:
+  name: register-seed
+spec:
+  owner: gardener-oq@listserv.sap.com
+  description: Tests the registration of a seed.
+  activeDeadlineSeconds: 7200
+
+  command: [bash, -c]
+  args:
+  - >-
+    go test -timeout=0 -mod=vendor ./test/system/seed_registration
+    --v -ginkgo.v -ginkgo.progress -ginkgo.noColor
+    -kubecfg=$TM_KUBECONFIG_PATH/gardener.config
+    -seed-name=$SEED_NAME
+    -shoot-kubecfg=$TM_KUBECONFIG_PATH/shooted-seed.config
+    -secret-binding=$SECRET_BINDING
+    -shoot-name=$SHOOT_NAME
+    -shoot-namespace=$PROJECT_NAMESPACE
+    -backup-secret-provider=$BACKUP_SECRET_PROVIDER
+    -provider-type=$PROVIDER_TYPE
+    -provider-region=$PROVIDER_REGION
+    -networking-default-shoot-services=$NETWORKING_DEFAULT_SERVICES
```

Comment:
Currently spec.settings.scheduling.visible is set to false in buildSeedObject func. What is the rationale behind making the seed invisible (no Shoots can be scheduled on it) and specifying spec.networks.shootDefaults (something that is used only when Shoots get scheduled on the Seed)?

Reply:
You're right the shoot defaults can be skipped as they are an optional parameter.

Reply:
Should we made it optional in the code or remove it completely ?

Reply:
Well, for simplicity let's remove them.